Plans/packit-ci.fmf

```diff
@@ -9,7 +9,7 @@ prepare:
      - dnf makecache
 
 environment:
-  TANG_IMAGE: "quay.io/sec-eng-special/fedora_tang_server"
```

Comment:
This should be unchanged. This is the Tang image (the Tang container), not the Tang operator container